great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -297,23 +358,148 @@ def compute_bootstrap_quantiles(
     computing the stopping criterion, expressed as the optimal number of bootstrap samples, needed to achieve a maximum
     probability that the value of the statistic of interest will be minimally deviating from its actual (ideal) value.
     """
+    lower_quantile_pct: float = false_positive_rate / 2
+    upper_quantile_pct: float = 1.0 - false_positive_rate / 2
+
+    sample_lower_quantile: Number = np.quantile(metric_values, q=lower_quantile_pct)
+    sample_upper_quantile: Number = np.quantile(metric_values, q=upper_quantile_pct)
+
     bootstraps: np.ndarray = np.random.choice(
         metric_values, size=(n_resamples, metric_values.size)
     )
-    lower_quantile = np.mean(
-        np.quantile(
-            bootstraps,
-            q=false_positive_rate / 2,
-            axis=1,
-            interpolation="linear",  # can be omitted ("linear" is default)
+
+    bootstrap_lower_quantiles: Union[np.ndarray, Number] = np.quantile(
+        bootstraps,
+        q=lower_quantile_pct,
+        axis=1,
+    )
+    bootstrap_lower_quantile_point_estimate: Number = np.mean(bootstrap_lower_quantiles)
+    bootstrap_lower_quantile_standard_error: Number = np.std(bootstrap_lower_quantiles)
+    bootstrap_lower_quantile_bias: Number = (
+        bootstrap_lower_quantile_point_estimate - sample_lower_quantile
+    )
+
+    # Bias / Standard Error > 0.25 is a rule of thumb for when to apply bias correction.
```

Comment:
@NathanFarmer Is this standard error or standard deviation?

Reply:
@alexsherstinsky the bootstrap sample standard deviation is referred to as the standard error in this context. See:
```
Efron, B., & Tibshirani, R. J. (1993). Standard errors and estimated standard errors. An Introduction to the Bootstrap (pp. 40).
        Springer Science and Business Media Dordrecht. DOI 10.1007/978-1-4899-4541-9
```